src/CMakeLists.txt

```diff
@@ -89,7 +89,6 @@ set (NNG_SRCS
 
 if (NNG_PLATFORM_POSIX)
     find_package (Threads REQUIRED)
-    set(NNG_LIBS Threads::Threads)
```

Comment:
This should turn into an append, not be removed entirely.  (The removal is working because we also pick up on -lpthread elsewhere.)